src/constants.ts

```diff
@@ -57,11 +58,10 @@ export const MULTICALL_ADDRESS = "0xcA11bde05977b3631167028862bE2a173976CA11";
 
 export const REVERSE_REACTOR_MAPPING: ReverseReactorMapping = Object.entries(
   REACTOR_ADDRESS_MAPPING
-).reduce((acc: ReverseReactorMapping, [chainId, orderTypes]) => {
+).reduce((acc: ReverseReactorMapping, [_, orderTypes]) => {
   for (const [orderType, reactorAddress] of Object.entries(orderTypes)) {
     // lowercase for consistency when parsing orders
     acc[reactorAddress.toLowerCase()] = {
-      chainId: parseInt(chainId),
```

Comment:
This breaks if we have the same reactor address on multiple chains. So removing the chainId here and requiring it as a param for the `parseOrder` function.